### PR TITLE
Miscellaneous VCF formatting and typo fixes

### PR DESCRIPTION
**Pull request by jmarshall**
Fix the BCF encoding errors noted in #382, and various formatting improvements.

No changes to the format or descriptions, so this should be straightforward to review and merge.

The non-TeX quotes etc in §1.4.7 _Contig field format_ are left untouched so as not to conflict with lines changed in PR #379.

### CODE REVIEW

File: VCFv4.3.tex [hunk 1]
```diff
@@ -284,55 +284,69 @@ \subsubsection{Fixed fields}
...
   Multiple bases are permitted.
   The value in the POS field refers to the position of the first base in the String.
-  For simple insertions and deletions in which either the REF or one of the ALT alleles would otherwise be null/empty, the REF and ALT Strings must include the base before the event (which must be reflected in the POS field), unless the event occurs at position 1 on the contig in which case it must include the base after the event; this padding base is not required (although it is permitted) for e.g. complex substitutions or other events where all alleles have at least one base represented in their Strings. 
+  For simple insertions and deletions in which either the REF or one of the ALT alleles would otherwise be null/empty, the REF and ALT Strings must include the base before the event (which must be reflected in the POS field), unless the event occurs at position 1 on the contig in which case it must include the base after the event; \pagebreak[1] this padding base is not required (although it is permitted) for e.g.\ complex substitutions or other events where all alleles have at least one base represented in their Strings.
```
[the remark this answers is not in the document]
Comment (jmarshall) [reply VCFv4.3.tex]: From the commit message: _encourage a pagebreak in "4 REF" to get more vertical space for Table 1 on page 8_. Without this, there is wasted space on the previous page and Table 1 is split more awkwardly.

(Also just pushed one more fix.)

**Comment by jmarshall** [comment]
Thanks. I would have merged that with a merge commit, as it's a sequence of distinct commits that tell a story. Squashing them into a single commit makes it a bit harder for future us to look back and understand the history, but YMMV.

**Comment by cyenyxe** [comment]
I actually prefer merge commits, but I thought the standard policy was always squashing...

**Comment by jmarshall** [comment]
I don't think we have a policy as such.

What often happens in this repository is that a single-commit PR gets improved and workshopped and acquires a bunch of “Fixed typos” and “Respond to review comments“ commits. Clearly the best practice in that case is to squash, as the fixup commits are not of later historical interest.

Multi-commit PRs containing distinct fixes are a different case.

**Pull request merged**